src/locale/languages/zhCn.js

```diff
@@ -89,6 +89,7 @@ module.exports = {
   'No contributors found': '没有贡献者',
   'No description provided.': '没有提供简介',
   'No issues': '没有问题',
+  'No members found': '',
```

Comment:
@chinesedfan can you provide a translation here?

Reply:
@lex111 Done. And thanks for your translations.